src/db/tripledb.plt

```diff
@@ -1,54 +1,71 @@
 :- begin_tests('tripledb').
-
 :- use_module(library('db/tripledb'),
     [ tripledb_load/2,
       tripledb_ask/3,
       tripledb_forget/3,
       tripledb_tell/3
     ]).
 
+:- use_module('../../../rosprolog/src/rostest.pl').
```

Comment:
use_module(library('rostest.pl')).

Reply:
This solution results into an error: source_sink `library(rostest)' does not exist.

Reply:
well pointing with a relative path to a file in another project is definitely not ok.

Probably one should make sure that the rosprolog src dir is registered as prolog library directory. That goes somewhat like that:

    asserta(user:library_directory(AbsoluteDirectory)),
    assert(user:file_search_path(ros, AbsoluteDirectory)).

Reply:
Do we need to register that in Knowrob project? Since it is already defined there in rosprolog project.